stable/opsportal/templates/ingress-opsportal.yaml

```diff
@@ -5,9 +5,7 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: traefik
     traefik.frontend.rule.type: PathPrefixStrip
-    ingress.kubernetes.io/auth-secret: ops-portal-htpasswd
-    ingress.kubernetes.io/auth-type: basic
-    ingress.kubernetes.io/auth-header-field: X-WebAuth-User
+    ingress.kubernetes.io/auth-header-field: X-Forwarded-User
```

Comment:
Opsportal is going to be replaced by konvoy-ui (based on kommander chart, yes, the same chart with different configurations). So I won't bother do any change in this chart.